packages/sdks/snippets/react/src/components/CustomColumns.tsx

```diff
@@ -5,30 +5,23 @@ import {
 } from '@builder.io/sdk-react';
 
 interface CustomColumnsProps {
-  columns: { blocks: BuilderBlock[] | undefined }[];
+  column1: { blocks: BuilderBlock[] | undefined };
+  column2: { blocks: BuilderBlock[] | undefined };
   builderBlock: BuilderBlock;
 }
 
 const CustomColumns = (props: CustomColumnsProps) => {
   return (
-    <div
-      style={{
-        display: 'flex',
-        flexDirection: 'row',
-        justifyContent: 'space-around',
-        border: '10px solid #ccc',
-        padding: '10px',
-      }}
-    >
+    <div>
```

Comment:
if i recall, this div was removed in the custom columns for angular, no? we should keep them consistent and make this a fragment